.readthedocs.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
```

Comment:
This looks like it differs from the template repo... what's the end result for this setup across all projects? We should maybe start there.